scripts/tls-common.sh

```diff
@@ -129,3 +133,32 @@ if ls "${BMC_CACERTS_PATH}"/* > /dev/null 2>&1; then
 else
     export BMC_TLS_ENABLED="false"
 fi
+
+if ls "${IPA_CACERTS_PATH}"/* > /dev/null 2>&1; then
+    cat "${IPA_CACERTS_PATH}"/* > "${IPA_CACERT_FILE}"
+else
+    copy_atomic "${DEFAULT_CACERT_BUNDLE}" "${IPA_CACERT_FILE}"
+fi
+
+if [ -f "${IRONIC_CACERT_FILE}" ]; then
+    cat "${IRONIC_CACERT_FILE}" >> "${IPA_CACERT_FILE}"
+fi
+
+generate_cacert_bundle_initrd()
+(
+    local output_path="$1"
+    local temp_dir
+
+    temp_dir="$(mktemp -d)"
+
+    cd "${temp_dir}" || return
+
+    mkdir -p etc/ironic-python-agent.d etc/ironic-python-agent
+    cat > etc/ironic-python-agent.d/ironic-tls.conf <<EOF
+[DEFAULT]
+cafile = /etc/ironic-python-agent/ironic.crt
+EOF
+    cp "${IPA_CACERT_FILE}" etc/ironic-python-agent/ironic.crt
+
+    find . | cpio -o -H newc --reproducible >> "${output_path}"
```

Comment:
Just checking.
If I remember correctly you have mentioned this only works with an initrd that is compressed with "zstd" or some other specific compression method right?

Reply:
Not this one, the way we currently do it at SUSE in the IPA downloader only works with "zstd" or other compression methods where you can simply append to the file.
Here it is a different initrd file that is loaded by PXE in addition to the IPA one (I do not compress it actually since I don't think it would be relevant, but we could also compress it here if needed)